### DISCUSSION TITLE
Sainsmart PROver and Lasergrbl

### DISCUSSION THREAD

**Discussion by Lobob48** [arkypita/LaserGRBL#1164] [Q&A]
HI all from a humble Newby,

Does anyone have a working Sainsmart 3018 PROver with 5500 Laser Module?
I've not been able to get the Laser to fire off by using the Lasergrbl software.  Lasergrbl connect just 
fine, I have contorl of axis and homing etc..  I've installed all of the custom buttons but laser focus, blink, and fan do
not encourage the laser to do anything.  The laser will fire if I lock in the PMW switch on the module.
I've tired several different $ settings and I do have the $32 =1 (laser enable).  Here's a picture of the
connections from module to laser and main board.  Any advice appreciated.

![20201107_164916](https://user-images.githubusercontent.com/74158565/98486463-9fa65f80-21da-11eb-8c1b-bc2484c23169.jpg)

**Comment by rwaynem53** [comment]
I'm having same issue. No answers yet. Don't know what to do. Sainsmart confirmed I was connected correctly but can't help with lasergrbl.
